### PR TITLE
Updates to Examiner Dashboard

### PR DESCRIPTION
**Pull request by dimak1**
*Issue:*

*Description of changes:*
- Updates to Examiner Dashboard based on wip designs

#### Dashboard

Screenshot 2024-12-19 at 15 04 18

#### Details / Examine

Screenshot 2024-12-19 at 15 04 27

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: strr-examiner-web/app/components/RegistrationDetails.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+const { header } = toRefs(props.application)
+
+</script>
+<template>
+  <ConnectPageSection
+    v-if="header?.registrationNumber"
+    :heading="{
+      label: 'Registration Details',
+      labelClass: 'font-bold',
+      icon: 'mdi-file-document-tick-outline'
```
Comment (deetz99) [inline strr-examiner-web/app/components/RegistrationDetails.vue:14]: icon should have 'i-' prefix, so this should be 'i-mdi-file-document-tick-outline'

**Review by deetz99 — APPROVED** [review]
Looks good 👍

File: strr-examiner-web/app/components/RegistrationDetails.vue (hunk 1 above)
Comment (dimak1) [reply strr-examiner-web/app/components/RegistrationDetails.vue]: Updated

File: strr-examiner-web/app/components/ApplicationDetails.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+const props = defineProps<{ application: ApiApplicationResp, applicationActions?: HeaderAction[] }>()
+
+const displayApplicationType = {
+  [ApplicationType.HOST]: 'Host',
+  [ApplicationType.PLATFORM]: 'Platform',
+  [ApplicationType.STRATA_HOTEL]: 'Strata Hotel'
+}
+
+const { header, registration } = toRefs(props.application)
```
[the remark this answers is not in the document]
Comment (dimak1) [reply strr-examiner-web/app/components/ApplicationDetails.vue]: Yea it is not needed. The component itself is not needed and will be cleaned up.

**Pull request merged**